plugins/generic/htmlArticleGalley/HtmlArticleGalleyPlugin.inc.php

```diff
@@ -180,6 +180,8 @@ function _getHTMLContents($request, $galley) {
 		foreach ($paramArray as $key => $value) {
 			$contents = str_replace('{$' . $key . '}', $value, $contents);
 		}
+		
+		HookRegistry::call('HtmlArticleGalleyPlugin::htmlGalleyContent', array(&$galley, &$contents));
```

Comment:
Let's pass the `$submissionFile` and `$embeddableFiles` and `$issue` with the hook. That way, we can save extra hits to the database to retrieve data that's already been retrieved.